### DISCUSSION TITLE
Not displaying webpage (UI) 404 error "file not found" Where did it go?

### DISCUSSION THREAD

**Discussion by Radius2023** [thehighnibble/thehighnibble.github.io#83] [Q&A] [answered]
Not displaying webpage (UI) 404 error "file not found" Where did it go?
 This was a Drag&drop Upgrade  from Ver. 4.5 (4.36) a late Fall 2019  
kit firmware to version 1.10. Currently I see the ESP32 webserver is alive
and broadcasting imsa8080 either in AP or STA mode. I can also scan 
active networks and see imsi8080 in my scan list. It seems I can't fix
this problem by going back to my original Micro-sd with ver. 4.36. New microsd card formatted as fat32 with a current image of 1.10. I am able to boot to a serial tty display of cp/m, Xybasic, receive an A> prompt, display directories, and run .com files. Not having the WebUI makes this machine pretty  limited.
.
I have included screenshots of NVS-log-info, unzipped image files, Tera-term serial tty connection, boot image, network broadcast and Google web browser login page. Will be excited to get back to the Web UI . If you have a solution, please comment.
### Network configuration
NTP_SERVER=pool.ntp.org
TZ=AEST-10AEDT,M10.1.0,M4.1.0
HOSTNAME=imsai8080
PORT=80
SSID=CenturyLink9388
PASSWORD=------------
WIFI.sta.scan=1
#WIFI.password.hide=1
### Enable S-132 
#S132.init=1
### UART configuration
#UART0=115200,cs8
#UART1=115200,cs8
### Performance parameters
TTY.netsrv.buffer_delay=33
LPT.netsrv.buffer_delay=33
### Harddisk image
HARDDISK=hd-ws4
### SIO-2 default port mappings (HAL)
#SIO1.portA.device=WEBTTY,UART0
#SIO1.portB.device=VIOKBD
#SIO2.portA.device=WEBPTR,UART1
#SIO2.portB.device=MODEM
### Modem initialization string. Example:
#MODEM.init=ATS0=1S15=1&A1 #S0=1 auto-answer after 1 ring, S15=1 enable telnet protocol
MODEM.init=ATS15=1 #S15=1 enable telnet protocol
![2023-06-28 use forum](https://github.com/thehighnibble/thehighnibble.github.io/assets/132217595/ad5f2fe9-b05c-4911-8336-57504529e02e)
![2023-07-03 use forum (2)](https://github.com/thehighnibble/thehighnibble.github.io/assets/132217595/b3d6af74-2efc-4a7f-8485-1ffb56ac6a53)
![2023-07-03 use forum](https://github.com/thehighnibble/thehighnibble.github.io/assets/132217595/d670c393-685f-4f87-8d1d-5c8ccf3d247b)
![2023-07-use forum  preamble  -log](https://github.com/thehighnibble/thehighnibble.github.io/assets/132217595/3a8fb74e-3ea4-4abc-95ee-8d63803d6ca2)
![2023-06-26 (3)](https://github.com/thehighnibble/thehighnibble.github.io/assets/132217595/514f2a12-d1e3-4910-a352-355e78399961)
![2023-06-26 (4)](https://github.com/thehighnibble/thehighnibble.github.io/assets/132217595/fd9759ac-6b8a-43bc-93ab-2d6a45ba6614)
![2023-06-26 use forum](https://github.com/thehighnibble/thehighnibble.github.io/assets/132217595/dbae1dbf-e7cb-48a9-af13-848778611cb8)
![2023-06-26](https://github.com/thehighnibble/thehighnibble.github.io/assets/132217595/7264dd22-9de3-4724-a714-c91939a80f3b)
![2023-06-27 - Copy](https://github.com/thehighnibble/thehighnibble.github.io/assets/132217595/1401e154-3e45-4c43-b7f8-37aa58f87f25)

**Comment by thisspaceleftblank** [comment]
I believe for 1.10 a lot changed and it requires rewriting the sdcard. I think that was when the Cromemco sim was introduced and everything got split into subdirectories..

**Reply by Radius2023** [reply]
08/19/2023

Configuring the old image file. I will paste the image soon.

**Comment by inerlogic** [comment]
just had the same issue after only updating the esp firmware. popped out the microSD card, copied over the image files, modified the config with my wifi settings, poof, works no problem.
from there i updated to 1.12 with no issues.

**Reply by Radius2023** [reply]
08/21/2023
This is what I have, (See attached) IMAGE V.1.10 Unzipped to a new fat32 Usd card.
This Sys code popped up twice while booting to Tera term. E<76781> 3cf3fd3b System op code trap.
Still no Joy! (No UI)
![2023-08-21](https://github.com/thehighnibble/thehighnibble.github.io/assets/132217595/eb736bd8-cd58-4502-b188-660740889a84)

**Comment by Radius2023** [comment, marked as answer]
Configuring the old image file. I will paste the image soon.

**Reply by Radius2023** [reply]
08/21/2023
 This is what I have, (See attached) IMAGE V.1.10 Unzipped to a new fat32 Usd card.
This Sys code popped up twice while booting to Tera term. E<76781> 3cf3fd3b System op code trap.
Still no Joy! (No UI)
![2023-08-21](https://github.com/thehighnibble/thehighnibble.github.io/assets/132217595/f589a459-a420-46ed-80db-d67968e8fccd)
